src/SourceLocator/Type/AutoloadSourceLocator.php

```diff
@@ -249,7 +225,10 @@ private function locateConstantByName(string $constantName) : ?string
 
         $constantFileName = null;
 
-        foreach (get_included_files() as $includedFileName) {
+        // Note: looking at files in reverse order, since newer files are more likely to have
```

Comment:
This changes the test run from ~7s per test to ~1s per test related to constant loading :muscle: